README.md

```diff
@@ -317,153 +352,178 @@ Step-by-step for those who need it
+| ------------- | ---------------------------------------------------------------- |
+| LE_HOSTNAME   | the hostname you are using to host the reverse proxy             |
+| LE_EMAIL      | the email for which your letsencrypt SSL cert will be registered |
+| LE_SUBDOMAINS | the subdomains for which to register the SSL cert                |
 
-## VPN networks for transmission, nzbget, and others
+### VPN networks for transmission, nzbget, and others
 
-I have built a wireguard container following the principles described [here](https://nbsoftsolutions.com/blog/routing-select-docker-containers-through-wireguard-vpn) and have successfully tested using docker-compose to force all traffic for `nzbget` through the wireguard VPN using a config from http://mullvad.net/.
+I have built a wireguard container following the principles described [here](https://nbsoftsolutions.com/blog/routing-select-docker-containers-through-wireguard-vpn) and have successfully tested using docker-compose to force all traffic for `nzbget` through the wireguard VPN using a config from <http://mullvad.net/>.
 
-Download or build a wireguard config file from your VPN provider. For example, mullvad has a wireguard config generator at https://mullvad.net/en/download/wireguard-config/. Once you have the config file (e.g. mine is called `wgnet0.conf`, copy that file into your config/wireguard directory. The container will automatically find the config file and build the VPN network for you.
+Download or build a wireguard config file from your VPN provider. For example, mullvad has a wireguard config generator at <https://mullvad.net/en/download/wireguard-config/>. Once you have the config file (e.g. mine is called `wgnet0.conf`, copy that file into your config/wireguard directory. The container will automatically find the config file and build the VPN network for you.
 
 ⚠️ Do NOT use your VPN's killswitch in the `wireguard` config file. This will disallow traffic from the docker network which will make it inaccessible from the reverse proxy container. There is a rudimentary killswitch implemented in the `wireguard` container already. I may improve this over time.
 
 `docker-compose.yml` has been updated with a commented out wireguard service. There is also a commented out `nzbget` service configured to use the wireguard container's network as an example. This will force `nzbget` to proxy all traffic through the `wireguard` container, and thus through the VPN. You can follow the same pattern with the `transmission` container, or even `sonarr` and `radarr` if you want all queries to torrent trackers to go through the VPN.
 
 To proxy an additional container through the `wireguard` container's network, add the service name (e.g. `transmission`) to the list of network aliases defined in the `wireguard` service.
 
-```
-    networks:
-      default:
-        aliases:
-          - nzbget
-          - transmission
+```yaml
```

Comment:
Could we actually space this whole block in four spaces like we have with the filebot section below?

Reply:
That would be an markdown linting issue. The four leading spaces are only needed if you want to add snippet in some sort of list - like below `1. 2. 3.`.